appendices/scope.md

```diff
@@ -0,0 +1,72 @@
+## Python package domain scope
+
+The following categories are the current domain areas that fall into the 
+pyOpenSci domain scope.
+
+- **Data retrieval:** Packages for accessing and downloading data from online sources. Includes wrappers for accessing APIs.
+- **Data extraction:** Packages that aid in retrieving data from unstructured sources such as text, images and PDFs.
+- **Data munging:** Tools for processing data from scientific data formats.
+- **Data deposition:** Tools for depositing data in scientific research repositories.
+- **Reproducibility:** Tools to scientists ensure that their research is reproducible. E.g. version control, automated testing, or citation tools.
+- **Geospatial:** Packages focused on the retrieval, manipulation, and analysis of spatial data.
+- **Education:** Packages to aid with instruction.
```

Comment:
@arianesasso education is also here because of me :) we could totally revisit this list maybe in a separate PR? discuss it first in an issue if you'd like to open an issue about it.

Reply:
Yes, I will do that! I also saw that in a different thread, so I guess it is time 😄 .

Reply:
plus one!

This also came up in https://github.com/pyOpenSci/python-package-guide/issues/21 -- maybe tag that issue too if you open a new one? 

I think the person opening it didn't realize it was on python-packaging-guide instead of peer-review-guide

They specifically talk about the way we list categories now possibly limiting what people might consider in scope
https://github.com/pyOpenSci/python-package-guide/issues/21#issuecomment-1368051779

In addition to revisiting categories, maybe it's worth thinking about what language we use to *introduce* this section.
Maybe we want to say something like "below is a list of some categories that fall within scope of pyOpenSci and our overall mission of enabling open science with Python. This list is not meant to be exclusive!"

... but I'll save that discussion for a new issue, don't mean to add noise to this review